tests/test_rds.py

```diff
@@ -610,8 +610,95 @@ def test_rds_db_subnetgroup_unused(self):
 
         self.assertEquals(len(resources), 1, "Resources should be unused")
 
+
 class RDSSnapshotTest(BaseTest):
 
+    def test_rds_snapshot_copy_tags_enable(self):
```

Comment:
rds as an api is pretty verbose, it would be several thousand lines cheaper in terms of recorded data to just use an rds client to verify pre/post condition, and in general thats preferrable to running policies for the same purpose. ie. we want to test a policy/filter or action, using a policy to verify that is typically wasteful vs using a client api call.